docs/src/main/asciidoc/building-native-image-guide.adoc

```diff
@@ -85,7 +85,7 @@ If you have generated the application from the previous tutorial, you can find i
 </profiles>
 ----
 
-We use a profile because, you will see very soon, packaging the native image takes a _few_ seconds.
+We use a profile because, you will see very soon, packaging the native executable takes a _few_ seconds.
```

Comment:
You didn't fix the issue with the filename of the runner?

We really need to have consistency between:
- I run using the guide instructions
- I get the solution from the quickstarts repo

Reply:
Forgot that one! Will fix it in the same PR.

Reply:
So, I thought about it and came up with a third solution - delete everything ;-)

With the current content of the generated projects; no need to have 3 sub-projects in quickstarts, only a single one is required. It avoids the issue and makes things much more fluid. 

This is done in: https://github.com/quarkusio/quarkus-quickstarts/pull/87